components/feature-tabs.js

```diff
@@ -5,22 +5,41 @@ import mui, { Tab, Tabs } from 'material-ui';
 
 class FeatureTabs extends BaseComponent {
   constructor () {
-    super()
+    super();
+
+
+    this.state = {
+        tabIndex: this.getTabIndex()
+    };
+
+    window.addEventListener("hashchange", this.onHashChange.bind(this), false);
   }
-  render () {
+
+  onHashChange () {
+    this.setState({
+        tabIndex: this.getTabIndex()
+    });
+  }
+
+  getTabIndex () {
     // Till material-ui gives an option to update tabIndex programatically.
     const tabIndex = (window.location.href.indexOf("#ES7")>=0 ? 0 :
```

Comment:
We don't need this anymore, right? But without the router, it's bit of a pain to generate the sitemap.